api/pom.xml

```diff
@@ -18,7 +18,7 @@
         <dependency>
             <groupId>org.bahmni.module</groupId>
             <artifactId>bahmni-migrator</artifactId>
-            <version>0.92-SNAPSHOT</version>
+            <version>${bahmniVersion}</version>
```

Comment:
I think the variable is better named as bahmniJavaUtils version, no?  this is about the libs from bahmni-java-utils, right?

Reply:
Yes @angshu, we changed variable name to bahmniJavaUtils, as it is pointing to libs from bahmni-java-utils